config/default/kustomization.yaml

```diff
@@ -28,7 +28,7 @@ patchesStrategicMerge:
 images:
 - name: kusk-gateway
   newName: kubeshop/kusk-gateway
-  newTag: v1.3.20
+  newTag: v1.3.22
```

Comment:
The latest release we've got is v1.3.19 https://github.com/kubeshop/kusk-gateway/tags
So leave v1.3.20

Reply:
Can I leave it as `v1.3.22`?

The reason I am changing it is because `v1.3.20` is the current release.

Reply:
then it should be v1.3.21

Reply:
Resolved.

Now at `v1.3.21` in latest commit.